curriculum/challenges/english/07-scientific-computing-with-python/learn-tree-traversal-by-building-a-binary-search-tree/65c645b838e7deb080fc25e0.md

```diff
@@ -1,62 +1,87 @@
 ---
-id: 655a4614304cd36031cb4e75
+id: 65c645b838e7deb080fc25e0
 title: Step 28
 challengeType: 20
 dashedName: step-28
 ---
 
 # --description--
 
-The deletion operation might result in a new root (for example, if the node to be deleted is the current root).
+Now, the nodes have been inserted. To see if they have  been correctly inserted, you can search for a node in the tree.
 
-To handle this case, assign the result of the `_delete` call to `self.root`.
+You can search for a node in the tree like this:
+
+```python
+your_object.search(node)
+```
```

Comment:
This is step 28 of the 10th project, is it really necessary to remind the camper how to call a method?